src/Umbraco.Web/Models/Mapping/MemberMapperProfile.cs

```diff
@@ -1,154 +1,185 @@
 ﻿using System;
+using System.Collections.Generic;
 using System.Web.Security;
-using AutoMapper;
 using Umbraco.Core;
+using Umbraco.Core.Mapping;
 using Umbraco.Core.Models;
+using Umbraco.Core.Models.Membership;
+using Umbraco.Core.Security;
 using Umbraco.Core.Services;
 using Umbraco.Web.Models.ContentEditing;
 using Umbraco.Core.Services.Implement;
+using UserProfile = Umbraco.Web.Models.ContentEditing.UserProfile;
 
 namespace Umbraco.Web.Models.Mapping
 {
     /// <summary>
     /// Declares model mappings for members.
     /// </summary>
-    internal class MemberMapperProfile : Profile
+    internal class MemberMapperProfile : IMapperProfile
     {
-        public MemberMapperProfile(
-            MemberTabsAndPropertiesResolver tabsAndPropertiesResolver,
-            MemberTreeNodeUrlResolver memberTreeNodeUrlResolver,
-            MemberBasicPropertiesResolver memberBasicPropertiesResolver,
-            IUserService userService,
-            IMemberTypeService memberTypeService,
-            IMemberService memberService)
+        private readonly CommonMapper _commonMapper;
+        private readonly IMemberTypeService _memberTypeService;
+        private readonly TabsAndPropertiesMapper<IMember> _tabsAndPropertiesMapper;
+
+        public MemberMapperProfile(CommonMapper commonMapper, IMemberTypeService memberTypeService, ILocalizedTextService localizedTextService)
+        {
+            _commonMapper = commonMapper;
+            _memberTypeService = memberTypeService;
+
+            _tabsAndPropertiesMapper = new TabsAndPropertiesMapper<IMember>(localizedTextService);
```

Comment:
This should probably be a `MemberTabsAndPropertiesMapper`, this could very well be the reason we are not mapping the custom generic properties

Reply:
oh yes